pkg/printers/internalversion/describe_test.go

```diff
@@ -2248,6 +2249,7 @@ func TestDescribePodSecurityPolicy(t *testing.T) {
 			Name: "mypsp",
 		},
 		Spec: policy.PodSecurityPolicySpec{
+			AllowedUnsafeSysctls: []string{"kernel.*", ",net.ipv4.ip_local_port_range"},
```

Comment:
What's about forbidden sysctls?

Reply:
> ",net.ipv4.ip_local_port_range"

Why it's started from comma?

Reply:
I thought I'd test only one of those two since many of the other attributes seem to be left out as well (e.g. `FlexVolume`) and forbidden sysctls have the same display method as allowed unsafe ones. I can add them if you want, though.